src/lib/components/organisms/HeaderDetailPage.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  import HeaderContent from '../molecules/HeaderContent.svelte'
+  import HeaderImage from '../atoms/HeaderImage.svelte'
+</script>
+
+<div class="header-container">
```

Comment:
That's a lot of nested ```<div>```'s, is this really necessary or can you use a more semantic option instead..